templates/named.conf.erb

```diff
@@ -0,0 +1,29 @@
+options {
+          listen-on port 53 { any; };
+          directory       "/var/named";
+          dump-file       "/var/named/data/cache_dump.db";
+          statistics-file "/var/named/data/named_stats.txt";
+          memstatistics-file "/var/named/data/named_mem_stats.txt";
+          allow-query     { any; };
+          recursion yes;
+
+
+          dnssec-enable no;
+          dnssec-validation no;
+          dnssec-lookaside auto;
+
+          forward first;
+          forwarders {
+              <%= external_dns %>;
```

Comment:
Thanks for the link @ColonelPanicks. @WilliamMcCumstie do we need to do anything about the second point there:

> Create an empty file at /etc/named/metalware.conf so that the file exists and named doesn't fail out

I don't understand why `named` would fail though, as shouldn't this file always be rendered?